### DISCUSSION TITLE
The background picture on my page is not showing

### DISCUSSION THREAD

**Discussion by lenzner** [community/community#46879] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello,
I was wondering if anyone can help me solving an issue with ma github page. I want to display a background picture called "bg.jpg" but it does not show. It might be something related to the path in my css but I just cannot figure out what the problem is

I checked similar posts that related to case sensitivity of the picture file ending etc. but did not manage to solve the problem myself.

Any help would be much appreciated.

**Comment by radar-blip** [comment, marked as answer]
Its named `bg1.jpg` in your images folder. Try that. 👍🏻 

Actually, you have two different background images in your images folder. `bg1.jpg` and `bg.JPG` 
Just use the correct one.

**Reply by lenzner** [reply]
I know, I want to have bg.jpg as background (i.e., the picture of a moss), I specified the name accordingly but it is still not working. But thanks for the comment anyways

**Reply by radar-blip** [reply]
Okay. Your image is named bg.JPG 
If you rename the image to bg.jpg
it should work, uppercase and lowercase matters.

**Reply by lenzner** [reply]
thanks so much that was it. It is weird as the name displayed in lower case on my computer.

**Reply by radar-blip** [reply]
You're welcome. Happy to help!
